geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -2641,9 +2640,7 @@ private VersionedObjectList sendMsgByBucket(final Integer bucketId, PutAllPRMess
             retryTime.waitToRetryNode();
           }
           event.setPossibleDuplicate(true);
-          if (prMsg != null) {
-            prMsg.setPossibleDuplicate(true);
-          }
+          prMsg.setPossibleDuplicate(true);
```

Comment:
Is there a reason you reduced protection after adding it in elsewhere?

Reply:
As with the other removed null check in Oplog.java, `prMsg` should never be null in this method, so checking if it is null here simply causes LGTM to assume that it may be null everywhere and complain any time a method is called on it without first checking if it's null.